src/thread/mod.rs

```diff
@@ -12,7 +12,7 @@ use neon_runtime;
 use neon_runtime::raw;
 
 /// A Rust task that can be executed in a background thread.
-pub trait Task: Send + Sized {
+pub trait Task: Send + Sized + 'static {
     /// The task's result type, which is sent back to the main thread to communicate a successful result back to JavaScript.
     type Output: Send;
```

Comment:
Should this be `'static` as well?

Reply:
I think so too. Take a look at `std::thread::spawn`'s signature:

```rust
pub fn spawn<F, T>(f: F) -> JoinHandle<T> where
    F: FnOnce() -> T,
    F: Send + 'static,
    T: Send + 'static, 
```
The return type of the function is required to be `Send + 'static`, and that's the analogous type to `Output`.